step.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
```

Comment:
add `set -e` after this? -> to fail if can't switch to the specified `dir_to_check`